### PR TITLE
fix(administration): fixed bugs at administration when user update & add error case

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Remove the reset password form for external and API users during user update.
  - API User
  image
  - External User
  image

- Change the timing of API calls to close the modal after the call is made.
- Add error case when user update. (the same as the previous password)

### CODE REVIEW

File: apps/web/src/services/administration/iam/user/type.ts [hunk 1]
```diff
@@ -68,6 +68,12 @@ export const USER_STATE = Object.freeze({
     DISABLE: 'DISABLED',
 } as const);
 
+export const USER_BACKEND_TYPE = {
+    LOCAL: 'LOCAL',
+    EXTERNAL: 'EXTERNAL',
+    API: 'API',
+} as const;
+
```
Comment (yuda110) [inline apps/web/src/services/administration/iam/user/type.ts:76]: praise: 👍

File: apps/web/src/services/administration/iam/user/modules/UserManagementTable.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@
             :items="userPageState.users"
             :select-index="userPageState.selectedIndices"
             :fields="fields"
-            :sort-by.sync="sortBy"
+            sort-by.sync="name"
```
Comment (yuda110) [inline apps/web/src/services/administration/iam/user/modules/UserManagementTable.vue:13]: nitpick: I think there's no need for `sync`!

File: apps/web/src/services/administration/iam/user/modules/UserManagementTable.vue [hunk 3]
```diff
@@ -158,32 +158,32 @@ export default {
             .setSort('name', true)
             .setFiltersAsRawQueryString(vm.$route.query.filters);
 
+        const fields = computed(() => ([
```
Comment (yuda110) [inline apps/web/src/services/administration/iam/user/modules/UserManagementTable.vue:161]: question: This seems to be static, so why is it computed?

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Review by WANZARGEN — APPROVED** [review]

**Pull request merged**